templates/generator.html.twig

```diff
@@ -10,6 +10,12 @@
 {% set halfColumn = 'col col-lg-5 col-lg-offset-1 col-md-5 col-md-offset-1 col-sm-12 col-xs-12' %}
 
 {% block body %}
+    <style>
+        .pl-30  {
+            padding-left: 30px;
+        }
+    </style>
```

Comment:
This style here doesn't seem to change anything below - checked on Firefox & chrome

Reply:
It adds extra padding on the 'note' paragraph like [here ](https://imgur.com/a/DvlUIGa) to make it aligned on the left.